### PR TITLE
Extend erlcloud with Security Hub module that implements “Descibe” op…

### PR DESCRIPTION
**Pull request by hardi171**
DescribeHub functionality added for security hub module.

### CODE REVIEW

File: src/erlcloud_securityhub.erl [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    Params :: params(),
+    Result :: {ok, securityhub()}| {error, not_found} | {error, term()}.
+describe_hub(AwsConfig, Params) ->
+    Path = ["accounts"],
+    case request(AwsConfig, _Method = get, Path, Params) of
+        {ok, Response} ->
+            {ok, Response};
+        {error, {<<"ResourceNotFoundException">>, _Message}} ->
+            {error, not_found};
+        {error, {<<"InvalidAccessException">>, _Message}} ->
```
Comment (motobob) [inline src/erlcloud_securityhub.erl:39]: I would not wrap this one. let it fall through into `{error, Reason}`

**Review by motobob — COMMENTED** [review]
minor changes and go

File: src/erlcloud_aws.erl [hunk 2]
```diff
@@ -668,6 +668,11 @@ service_config( Service, Region, Config ) when is_atom(Region) ->
     service_config( Service, atom_to_binary(Region, latin1), Config );
 service_config( Service, Region, Config ) when is_list(Region) ->
     service_config( Service, list_to_binary(Region), Config );
+service_config( <<"security_hub">>, Region, Config ) ->
```
[the remark this answers is not in the document]
Comment (hardi171) [reply src/erlcloud_aws.erl:671]: done.

File: src/erlcloud_securityhub.erl (hunk 1 above)
Comment (hardi171) [reply src/erlcloud_securityhub.erl]: done.

File: src/erlcloud_aws.erl (hunk 2 above)
[the remark this answers is not in the document]
Comment (hardi171) [reply src/erlcloud_aws.erl]: done.

**Review by motobob — APPROVED** [review]

**Review by motobob — APPROVED** [review]

**Pull request merged**